### PR TITLE
TC_03.08.007 | +New Item > Copy from > Redirect to the new project page

### PR DESCRIPTION
**Pull request by gurinowicz**
https://trello.com/c/PBKmWT1k

### CODE REVIEW

**Comment by gurinowicz** [comment]
[TC_03.08.007 | +New Item > Copy from > Redirect to the new project page](https://trello.com/c/PBKmWT1k/147-tc0308007-new-item-copy-from-redirect-to-the-new-project-page)

**Review by ArtemChumachenko — APPROVED** [review]

File: cypress/e2e/redirectionNewProjectPage.cy.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/// <reference types="cypress"/>
+
+import newJobPage from "../fixtures/newJobPage.json"
+import newItem from "../fixtures/newItem.json"
```
Comment (KravchukKseniya) [inline cypress/e2e/redirectionNewProjectPage.cy.js:4]: What is the purpose of these imports?

File: cypress/e2e/redirectionNewProjectPage.cy.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+describe('redirectionNewProjectPage', () => {
+    it('TC_03.08.007 +New Item > Copy from > Redirect to the new project page', () => {
+        cy.visit('http://localhost:8080/view/all/newJob');
```
Comment (Rom-T) [inline cypress/e2e/redirectionNewProjectPage.cy.js:8]: visit() should not be used. Need steps that will take you to this page.

File: cypress/e2e/redirectionNewProjectPage.cy.js (hunk 1 above)
Comment (gurinowicz) [reply cypress/e2e/redirectionNewProjectPage.cy.js:4]: Want them to taken me to the needed page and forget about it. Thank you. Fix them

File: cypress/e2e/redirectionNewProjectPage.cy.js (hunk 2 above)
Comment (gurinowicz) [reply cypress/e2e/redirectionNewProjectPage.cy.js]: > visit() should not be used. Need steps that will take you to this page.

Hi! Should I use it with construction beforeEach?

File: cypress/e2e/redirectionNewProjectPage.cy.js (hunk 1 above)
Comment (Rom-T) [reply cypress/e2e/redirectionNewProjectPage.cy.js]: No, it’s better not to use visit(), but to register all the checks before this endpoint.

File: cypress/e2e/redirectionNewProjectPage.cy.js (hunk 2 above)
Comment (Rom-T) [reply cypress/e2e/redirectionNewProjectPage.cy.js]: No, it’s better not to use visit(), but to register all the checks before this endpoint.

File: cypress/e2e/redirectionNewProjectPage.cy.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+describe('redirectionNewProjectPage', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:8080/view/all/newJob')
+    });
+    it('TC_03.08.007 +New Item > Copy from > Redirect to the new project page', () => {
+        cy.get('input#name').type('NewProject');
+        cy.get('.com_cloudbees_hudson_plugins_folder_Folder').click();
+        cy.get('#ok-button').click();
+        cy.url().should('be.equal', 'http://localhost:8080/job/NewProject/configure')
```
Comment (KravchukKseniya) [inline cypress/e2e/redirectionNewProjectPage.cy.js:11]: I need to get rid of this hard code too. You can change your assertion like
`should('contain', '/job/NewProject/configure')`

Also, you can move your data, like '/job/NewProject/configure' to file in fixtures folder.

**Review by KravchukKseniya — DISMISSED** [review]

**Comment by Paul-Korz** [comment]
Hi. Please close your PR and delete branch.

**Pull request closed without merging**